tools/topology/topology2/platform/intel/sdw-amp-generic.conf

```diff
@@ -0,0 +1,84 @@
+# route and pipeline index start from pcm id * 10
+
+Define {
+	SDW_SPK_STREAM 'SDW1-Playback'
+	ALH_2ND_SPK_ID 22
+	SDW_AMP_BE_ID 2
+}
+
+Object.Dai {
+	ALH."514" {
+		id 		$SDW_AMP_BE_ID
+		direction	"playback"
+		name		$SDW_SPK_STREAM
+		default_hw_conf_id	0
+		rate			48000
+		channels		2
+
+		Object.Base.hw_config."ALH514" {
+			id	0
+		}
+	}
+}
+
+Object.Pipeline {
+	host-copier-gain-mixin-playback.20 {
+		index 20
+
+		Object.Widget.copier.1 {
+			stream_name "sdw amplifiers"
+		}
+		Object.Widget.gain.1 {
+			Object.Control.mixer.1 {
+				name 'Amplifier Volume'
+			}
+		}
+	}
+
+	mixout-gain-dai-copier-playback.21 {
+		index 21
+
+		Object.Widget.copier.1 {
+			stream_name $SDW_SPK_STREAM
+			dai_type "ALH"
+			copier_type "ALH"
+			node_type $ALH_LINK_OUTPUT_CLASS
+		}
+		Object.Widget.gain.1 {
+			Object.Control.mixer.1 {
+				name 'Amplifier Volume'
```

Comment:
this 'Amplifier Volume' is the same as above 'Amplifier Volume",    you can call it "main Amplifier Volume"

Reply:
@RanderWang @ranj063 The widget name will be added in the control name. So the two 'Amplifier Volume' controls are `gain.20.1 Amplifier Volume` and `gain.21.1 Amplifier Volume`. Do we need to add different prefix for them?

Reply:
gain.20 or gain21 are not meaning and will confuse user